resources/views/components/article_thumbnail.blade.php

```diff
@@ -1,16 +1,35 @@
+@php
+
+$url = $url ?? $article->getUrl();
+$showEduTags = $showEduTags ?? false;
+    
+@endphp
 <div class="article-thumbnail">
   @if ($article->main_image)
-    <a href="{!! $article->getUrl() !!}" class="image-container">
+    <a href="{{ $url }}" class="image-container">
       <img data-src="{!! $article->getThumbnailImage() !!}" class="img-responsive lazyload" alt="{{ $article->title }}">
     </a>
   @endif
-  <a href="{!! $article->getUrl() !!}">
+  <a href="{{ $url }}">
     <h4 class="title">
       {!! $article->title_with_category !!}
     </h4>
   </a>
-  <p class="attributes">{!! $article->getShortTextAttribute($article->summary, 250) !!}
-    (<a href="{!! $article->getUrl() !!}">{{ trans('general.more') }}</a>)
-  </p>
-  <p class="meta">{!!$article->published_date!!} / {!!$article->author!!}</p>
-</div>
\ No newline at end of file
+  <div class="attributes">{!! $article->getShortTextAttribute($article->summary, 250) !!}
+    (<a href="{{ $url }}">{{ trans('general.more') }}</a>)
+  </div>
+  @if($showEduTags)
+  <div class="mt-3">
+    @foreach ($article->edu_target_age_groups ?? [] as $ageGroup)
+      <a href="{{ request()->fullUrlWithQuery(['age_group' => $ageGroup]) }}" class="btn btn-default btn-xs btn-outline">{{ trans("edu.age_group.$ageGroup") }}</a>
+    @endforeach
+    @foreach ($article->edu_media_types ?? [] as $mediaType)
+      <a href="{{ request()->fullUrlWithQuery(['media_type' => $mediaType]) }}" class="btn btn-default btn-xs btn-outline">{{ trans("edu.media_type.$mediaType") }}</a>
+    @endforeach
+    @foreach ($article->edu_keywords ?? [] as $keyword)
+      <a href="{{ request()->fullUrlWithQuery(['keyword' => $keyword]) }}" class="btn btn-default btn-xs btn-outline">{{ $keyword }}</a>
+    @endforeach
```

Comment:
@eronisko what do you think about

```suggestion
    @foreach ($article->edu_target_age_groups ?? [] as $ageGroup)
      <a href="{{ route('frontend.educational-article.index', ['age_group' => $ageGroup]) }}" class="btn btn-default btn-xs btn-outline">{{ trans("edu.age_group.$ageGroup") }}</a>
    @endforeach
    @foreach ($article->edu_media_types ?? [] as $mediaType)
      <a href="{{ route('frontend.educational-article.index', ['media_type' => $mediaType]) }}" class="btn btn-default btn-xs btn-outline">{{ trans("edu.media_type.$mediaType") }}</a>
    @endforeach
    @foreach ($article->edu_keywords ?? [] as $keyword)
      <a href="{{ route('frontend.educational-article.index', ['keyword' => $keyword]) }}" class="btn btn-default btn-xs btn-outline">{{ $keyword }}</a>
    @endforeach
```

Reply:
No preference from me here... If both you and @ustengrav think it should behave like this, I'm happy for this to be changed 👍

Reply:
yes, I think this way is more consistent with how clicking on this kind of "tag" generally works on other websites